src/App.js

```diff
@@ -1,13 +1,17 @@
-import { Suspense, React } from 'react';
+import { Suspense, React, lazy } from 'react';
 import { Route, BrowserRouter as Router, Switch, Redirect } from 'react-router-dom';
 
 import { Spin } from 'antd';
```

Comment:
group imports like this and sort them alphabetically by their path

react
//space
3rd party imports
//space
in project absolute imports
//space
in project relative imports